pkg/cache/cache_test.go

```diff
@@ -842,81 +893,185 @@ func TestCacheWorkloadOperations(t *testing.T) {
 			},
 			wantResults: map[string]result{
 				"one": {
-					Workloads:     sets.NewString("a", "c"),
-					UsedResources: Resources{"cpu": {"on-demand": 10, "spot": 15}},
+					Workloads:     sets.NewString("a", "b", "d"),
+					UsedResources: Resources{"cpu": {"on-demand": 20, "spot": 30}},
 				},
 				"two": {
-					Workloads:     sets.NewString("b", "d", "e", "f"),
+					Workloads:     sets.NewString("c", "e"),
 					UsedResources: Resources{"cpu": {"on-demand": 10, "spot": 15}},
 				},
 			},
 			wantAssumedWorkloads: map[string]string{
-				"/a": "one",
-				"/f": "two",
+				"/d": "one",
```

Comment:
I don't know if this answers your question but I think that is because `d` and `e` are the only workloads we assume in this test case.

https://github.com/kubernetes-sigs/kueue/pull/262/files/3ed4ded71520eb702a898cf1bd4fd5d0653ab68d#diff-3e99be25ef623d0bc22234a878bef63a1fef2029a18d5b35f9079537e9f59ebeR867-R886